tests/integration-service/status-reporting-to-pullrequest.go

```diff
@@ -18,7 +18,7 @@ import (
 	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 )
 
-var _ = framework.IntegrationServiceSuiteDescribe("Status Reporting of Integration tests", Label("integration-service", "HACBS", "status-reporting"), func() {
+var _ = framework.IntegrationServiceSuiteDescribe("Status Reporting of Integration tests", Label("integration-service", "status-reporting"), func() {
```

Comment:
```suggestion
var _ = framework.IntegrationServiceSuiteDescribe("Status Reporting of Integration tests", Label("integration-service", "github-status-reporting"), func() {
```
since the gitlab-specific test suite has its own label too.

Reply:
agree, 
Done
thanks